cluster/README.md

```diff
@@ -0,0 +1,5 @@
+# Cluster Configuration
```

Comment:
Let's not copy in the README from the main cluster directory. If we have a README here, it might be better for it to explain how this cluster directory relates to and differs from the kubernetes cluster directory.